test/js/dropdown-utils_test.js

```diff
@@ -38,6 +38,40 @@ describe('Dropdown utils', function() {
     expect($('option')).to.have.length(4);
   });
 
+  it('should not add options with same values', function() {
+    dd('foo').addOption({label: 'first', value: 'bar'});
+    dd('foo').addOption({label: 'second', value: 'bar'});
+    expect($('option')).to.have.length(2);
+  });
+
+  it('should select the option when values.select', function() {
+    dd('foo').addOption({label: 'Foo', value: 'BAR', select: 1});
+    expect( $('#foo')[0].selectedIndex ).to.equal(1);
+  });
+
+  it('should create option with value/name of ""', function() {
+    dd('foo').addOption(null);
+    $('#foo')[0].selectedIndex = 1;
+    expect($('#foo').val()).to.equal('');
+  });
+
+  it('should disable several options', function() {
```

Comment:
I dunno if I can push to your repo..but here's my attempt to do it, still not the best....
https://github.com/fna/owning-a-home/compare/increase-test-coverage...amymok:fna-increase-test-coverage